rs/tests/driver/src/driver/resource.rs

```diff
@@ -390,14 +390,27 @@ fn vm_spec_from_nested_node(
     node: &NestedNode,
     default_vm_resources: Option<VmResources>,
 ) -> VmSpec {
+    let vm_resources = &node.vm_resources;
     VmSpec {
         name: node.name.clone(),
         // Note that the nested GuestOS VM uses half the vCPUs and memory of this host VM.
```

Comment:
```suggestion
        // Note that the nested HostOS VM uses half the vCPUs and memory of this host VM.
```
Isn't it HostOS that gets half the vCPUs? We write the deployment config containing half the vCPUs [here](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/ic_os/dev_test_tools/setupos-image-config/src/lib.rs?L127-128) and later read from that file [here](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/ic_os/config/src/main.rs?L141) setting HostOS's number of CPUs to that number.

Reply:
Ah right, thank you!